### PR TITLE
Update ruleset: Ubisoft Entertainment

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

File: src/chrome/content/rules/Ubisoft-Entertainment.xml [hunk 1]
```diff
@@ -1,50 +1,25 @@
...
-		- static7.ubi.com.s3.amazonaws.com
-
-			- static7.ubi.com
-
-		- ubisoft.hs.llnwd.net
-
-			- static[79].cdn.ubi.com
-
-
 	Nonfunctional subdomains:
```
Comment (J0WI) [inline src/chrome/content/rules/Ubisoft-Entertainment.xml]: `static.cdn.ubi.com`

File: src/chrome/content/rules/Ubisoft-Entertainment.xml [hunk 2]
```diff
@@ -1,50 +1,25 @@
...
 
 	Problematic subdomains:
-
-		- static7	(amazonws)
-		- video.us	(works; mismatched, CN: *.ubi.com)
-
+		- static13.cdn: Invalid certificate, no known CDN bucket
+		- static16: Invalid certificate, no known CDN bucket
 
 	Fully covered subdomains:
```
Comment (J0WI) [inline src/chrome/content/rules/Ubisoft-Entertainment.xml]: This section is obsolete

File: src/chrome/content/rules/Ubisoft-Entertainment.xml [hunk 3]
```diff
@@ -53,79 +28,67 @@ Fetch error: http://static16.ubi.com/ => https://s3.amazonaws.com/static16.ubi.c
 
 	ToDo: Secure shop pages that don't redirect to http.
```
Comment (J0WI) [inline src/chrome/content/rules/Ubisoft-Entertainment.xml]: This comment is obsolete, `store.ubi.com` can be added as target.

File: src/chrome/content/rules/Ubisoft-Entertainment.xml [hunk 4]
```diff
@@ -53,79 +28,67 @@ Fetch error: http://static16.ubi.com/ => https://s3.amazonaws.com/static16.ubi.c
...
-	
-	<rule from="^http://static(1|3|4|6|7|9|10|11|13|14|17)\.cdn\.ubi\.com/"
+	<!--
+		static*.cdn.ubi.com
+			- Handled by http→https rule: static(2|5|7).cdn
+			- Handled by akamaihd.net rule: static(3|7|9|10|11|14|17|19).cdn
+			- In problematic subdomains: static13.cdn
+	-->
+	<rule from="^http://static(3|7|9|10|11|14|17|19)\.cdn\.ubi\.com/"
 		to="https://ubistatic$1-a.akamaihd.net/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Ubisoft-Entertainment.xml]: Please add some test urls

File: src/chrome/content/rules/Ubisoft-Entertainment.xml [hunk 5]
```diff
@@ -53,79 +28,67 @@ Fetch error: http://static16.ubi.com/ => https://s3.amazonaws.com/static16.ubi.c
...
-		
-	<rule from="^http://static(7|15|16)\.ubi\.com/"
+	<!--
+		static*.ubi.com
+			- Handled by http→https rule: static(1|2|3|4|5|6|8|9|11|17|19)
+			- Handled by amazonaws.com rule: static(7|15)
+			- In problematic subdomains: static16
+	-->
+	<rule from="^http://static(7|15)\.ubi\.com/"
 		to="https://s3.amazonaws.com/static$1.ubi.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Ubisoft-Entertainment.xml]: Please add some test urls

File: src/chrome/content/rules/Ubisoft-Entertainment.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/Ubisoft-Entertainment.xml]: `www.uplay.ubi.com`

File: src/chrome/content/rules/Ubisoft-Entertainment.xml (hunk 3 above)
Comment (injust) [reply src/chrome/content/rules/Ubisoft-Entertainment.xml]: Fixed.

File: src/chrome/content/rules/Ubisoft-Entertainment.xml (hunk 1 above)
Comment (injust) [reply src/chrome/content/rules/Ubisoft-Entertainment.xml]: Both added.

File: src/chrome/content/rules/Ubisoft-Entertainment.xml (hunk 2 above)
Comment (injust) [reply src/chrome/content/rules/Ubisoft-Entertainment.xml]: Fixed.

File: src/chrome/content/rules/Ubisoft-Entertainment.xml (hunk 4 above)
Comment (injust) [reply src/chrome/content/rules/Ubisoft-Entertainment.xml]: Added.

File: src/chrome/content/rules/Ubisoft-Entertainment.xml (hunk 5 above)
Comment (injust) [reply src/chrome/content/rules/Ubisoft-Entertainment.xml]: Added.

File: src/chrome/content/rules/Ubisoft-Entertainment.xml [hunk 6]
```diff
@@ -62,32 +65,42 @@
...
 	<!--
-		static*.cdn.ubi.com
+		static*.cdn:
 			- Handled by http→https rule: static(2|5|7).cdn
-			- Handled by akamaihd.net rule: static(3|7|9|10|11|14|17|19).cdn
+			- Handled by akamaihd.net rule: static(|3|7|9|10|11|14|17|19).cdn
 			- In problematic subdomains: static13.cdn
 	-->
-	<rule from="^http://static(3|7|9|10|11|14|17|19)\.cdn\.ubi\.com/"
+	<rule from="^http://static(|3|7|9|10|11|14|17|19)\.cdn\.ubi\.com/"
```
Comment (J0WI) [inline src/chrome/content/rules/Ubisoft-Entertainment.xml]: This regex looks wrong. Did you mean `(3|7|9|10|11|14|17|19)?` ?

Comment (injust) [reply src/chrome/content/rules/Ubisoft-Entertainment.xml]: No, I needed to include `static.cdn.ubi.com` so I added the extra `|`.

Comment (injust) [reply src/chrome/content/rules/Ubisoft-Entertainment.xml]: My bad. Not too familiar with regex yet. Fixing!

**Pull request merged**